doc/source/examples/multiphysics/cylinder-cooling/cylinder-cooling.rst

```diff
@@ -0,0 +1,267 @@
+        set file name            = cylinder.msh
+    end
+
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+The next step is establishing the boundary conditions for the fluid flow. We specify an inlet velocity of 20 (id = 3) in order to obtain a Reynolds number of 20. A slip boundary condition is set on the outer walls of the domain (id = 4) and an outlet boundary conditions is set for the right boundary (id = 5). The ids of the boundaries are written in the second column of the ``cylinder.msh`` file in the ``$PhysicalNames`` section.
+
+.. code-block:: text
+
+    subsection boundary conditions
+        set number = 3
+
+        subsection bc 0
+            set id = 3
+            set type = function
+            subsection u
+            set Function expression = 20
+            end
+            subsection v
+            set Function expression = 0
+            end
+            subsection w
+            set Function expression = 0
+            end
+        end
+
+        subsection bc 1
+            set id = 4
+            set type = outlet
+        end
+
+        subsection bc 2
+            set id = 5
+            set type = slip
+        end
+    end
+
+Boundary Conditions Heat Transfer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the heat transfer side, we apply a constant temperature of 0 at the inlet (id = 3) to impose the cold temperature of the fluid. We explicitly specify ``noflux`` boundary conditions on the domain limits (id = 4) and on the outlet.
+
+.. code-block:: text
+
+    subsection boundary conditions heat transfer
+        set number = 3
+
+        subsection bc 0
+            set id    = 3
+            set type  = temperature
+            subsection value
+            set Function expression = 0
+            end
+        end
+
+        subsection bc 1
+            set id   = 4
+            set type = noflux
+        end
+
+        subsection bc 2
+            set id   = 5
+            set type = noflux
+        end
+    end
+
+Initial conditions
+~~~~~~~~~~~~~~~~~~~~
+
+The initial velocity is set to 20 in the x direction and 0 in the y and z directions. To specify the initial temperature throughout the domain, we use a conditional statement. If the node is inside the cylinder, the initial temperature is set to 1. Anywhere else, the initial temperature is set to 0.
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+
+        subsection uvwp
+            set Function expression = 20; 0; 0
+        end
+
+        subsection temperature
+            set Function expression = if(((x-8)^2+(y-0)^2)^0.5-0.5001<0,1,0) 
+        end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+Next, we define the physical properties for both the solid and the fluid. It is important to explicitly indicate the number of solids, otherwise, the solid region will not be detected by Lethe. The thermal conductivity is set so the Prandtl number is equal to 5. All the other physical properties are set to 1. 
```

Comment:
```suggestion
Next, we define the physical properties for both the solid and the fluid. It is important to explicitly indicate the number of solids, otherwise, the solid region will not be detected by Lethe. The thermal conductivity is set so the Prandtl number is equal to :math:`5`. All the other physical properties are set to :math:`1`. 
```